module/zfs/dmu.c

```diff
@@ -2483,7 +2467,6 @@ EXPORT_SYMBOL(dmu_free_long_object);
 EXPORT_SYMBOL(dmu_read);
 EXPORT_SYMBOL(dmu_read_by_dnode);
 EXPORT_SYMBOL(dmu_write);
-EXPORT_SYMBOL(dmu_write_by_dnode);
```

Comment:
Let's get @behlendorf's take on this.  It may be that Lustre is using this interface.

Reply:
Lustre is indeed using this function.  We're going to need to keep this, but it wouldn't be a bad idea to add a comment that Lustre is an external consumer of this interface and it's not unused.

Reply:
Done.